lib/matplotlib/artist.py

```diff
@@ -1385,6 +1439,46 @@ def aliased_name(self, s):
         aliases = ''.join(' or %s' % x for x in sorted(self.aliasd.get(s, [])))
         return s + aliases
 
+    _NOT_LINKABLE = {
+        # A set of property setter methods that are not available in our
+        # current docs. This is a workaround used to prevent trying to link
+        # these setters which would lead to "target reference not found"
+        # warnings during doc build.
+        'matplotlib.image._ImageBase.set_alpha',
```

Comment:
There's quite a few duplicates in this list?

Reply:
yup.